### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by pujagarwal**
Please review

### CODE REVIEW

File: swagger.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        type: "string"
+      - name: "API Key"  
+        in: "query"
+        description: "Enter API key"
+        required: true
+        type: "string"
+      responses:
+        "200":
+          description: "successful operation"
+          schema:
```
Comment (delyar) [inline swagger.yml:40]: Nice!

**Review by delyar — CHANGES_REQUESTED** [review]
This is a great job Puja! I just requested a change to test!

**Review by pujagarwal — COMMENTED** [review]
changed

File: swagger.yml (hunk 1 above)
Comment (delyar) [reply swagger.yml:40]: Thank you for reolving it

**Pull request merged**